tldw_Server_API/app/core/Setup/audio_pack_service.py

```
@@ -0,0 +1,259 @@
+    return json.dumps(payload, sort_keys=True, separators=(",", ":")).encode("utf-8")
+
+
+def _sha256_hexdigest(payload: bytes) -> str:
+    return hashlib.sha256(payload).hexdigest()
+
+
+def _copy_asset_manifest(installed_assets: list[dict[str, Any]] | None) -> list[dict[str, Any]]:
+    assets: list[dict[str, Any]] = []
+    for entry in installed_assets or []:
+        if isinstance(entry, dict):
+            assets.append(dict(entry))
```

Ruff (0.15.6)

[warning] Use a list comprehension to create a transformed list

(PERF401)